### ISSUE TITLE
Multi-machine learning rate setting

### ISSUE THREAD

**Issue by ecr23xx** [facebookresearch/SlowFast#271]
Hi, I have a question about multi-machine training lr and batch size setting.

For example, for one machine (8 gpus) training, the batch size is 64 and learning rate is 0.1. When I set NUM_SHARDS = 2, should I linearly scale the BASE_LR, BATCH_SIZE, and NUM_GPUs?

In my understanding of this code, it seems I only need to linearly scale the BASE_LR and keep BATCH_SIZE and NUM_GPUs unchanged?

**Comment by donnyyou** [comment]
Have you achieve the same accuracy with NUM_SHARDS=2? If so, could you share the settings you mentioned?

**Comment by ecr23xx** [comment]
When NUM_SHARDS is set to >1, only need to scale learning rate linearly and keep the batch size / num_gpus unchanged.
